cfpb/sbl-frontend
# feat(complete user profile - no assoc): added `/profile/complete/submitted` with summary result

closes #792 

## Changes
- feat(complete user profile - no assoc): added `/profile/complete/submitted` with summary result

## How to Test
### Testing Normal Use
1. Fill out a _Complete User Profile_ form with a user that has no associated institutions and verify that on submission 
2. Refresh the page and verify that the same summary should appear
3. While the summary is present, click the browser's `Back` button and then the `Forward` button to test the history
### Trying to break the PR
1. Navigate to the filing home and then enter the `/profile/complete/submitted` path **directly** in browser and verify that the user is routed to `/`

## Screenshots
|Filling Out|Submitted|
|--|--|
|<img width="1037" alt="Screenshot 2024-07-10 at 7 58 52 AM" src="https://github.com/cfpb/sbl-frontend/assets/13324863/4d591776-9b41-43ba-bff0-5f5ec0e671df">|<img width="1038" alt="Screenshot 2024-07-10 at 7 59 05 AM" src="https://github.com/cfpb/sbl-frontend/assets/13324863/2870fd25-c780-4f19-906e-7d6f0ae4564f">|
